packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -156,6 +163,18 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps & Cont
       <Stack.Item grow>
         <CallAdapterProvider adapter={props.callAdapter}>
           <Stack horizontalAlign="center">
+            {/*
+              hidden button to set first tab keypress focus on ControlBar on initial render. 
```

Comment:
This is an extremely creative way, nice job! Wondering if we could also use this as a common solution for CallComposite? Wondering if we could have a common solution across the composite which makes each piece more readable (I guess chat might have this problem in the future)

Would be nice to have a common component named HiddenFocusStartPoint (Or anything better) then put a button plus useEffects() there then well comment it, then this would be a nice common solution whenever we need it in composite

Reply:
I implemented your suggestion and it works AMAZINGLY well! 
Great idea Nan!! Really improves on the scalability and cleanliness of the code as well. 
I've submitted new changes with your suggestion. 

I have tested it extensively to make sure it is working in Call Composite and CallWithChat Composite desktop mobile and with voiceOver on to ensure accessibility is working correctly.

Reply:
Maybe we could directly apply this to callComposite as well in this PR I guess? Putting an element at the top of DOM tree will work for current case for sure, but we are working on a future proof solution - we can't ensure the element would always stay on the top, even it is reversed order(we might be adding more element around)

But thanks to your common solution, we have one-stop component to solve them all together

Reply:
You make a good point that we can't guarantee that the controlBar will always be the first interact-able element in the DOM tree. However, I believe it's good common practice to have tab focus order top down in the DOM Tree.

I'm okay with having the Call Composite use the same solution, but the solution for CallWithChat was only implemented as a workaround as reversing the order in CallWithChat did not resolve the focus issue.

Reply:
Either way works for me, keep it in best practice is also a good point to me. We just need to keep in mind that, we have an elegant single component implemented by you to solve this problem, and apply it whenever it is not possible to keep the element on the top

(Maybe just add some more comments on in HiddenFocusStartPoint like "This is a util component when we can't ensure the initial element for first tab focus is at the top of dom tree, and it moves the first-tab focus to be on the element right after it")

Reply:
We can definitely revisit this fix for CallComposite later down the line :)
For now, I will keep it in dom tree order. Thanks a lot Nan!!